src/bptl/work_units/kownsl/tests/tests_kownsl_tasks.py

```diff
@@ -0,0 +1,183 @@
+import copy
+import json
+
+from django.test import TestCase
+
+import requests_mock
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+
+from bptl.camunda.models import ExternalTask
+from bptl.tasks.models import TaskMapping
+from bptl.work_units.kownsl.tasks import (
+    get_client,
+    get_review_request,
+    get_review_request_reminder_date,
+    get_review_response_status,
+)
+from bptl.work_units.zgw.tests.factories import DefaultServiceFactory
+
+from .utils import mock_service_oas_get
+
+KOWNSL_API_ROOT = "https://kownsl.nl/"
+
+
+@requests_mock.Mocker()
+class KownslAPITests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        mapping = TaskMapping.objects.create(
+            topic_name="some-topic",
+        )
+
+        cls.service = Service.objects.create(
+            label="Kownsl",
+            api_type=APITypes.orc,
+            api_root=KOWNSL_API_ROOT,
+            auth_type=AuthTypes.api_key,
+            header_key="Authorization",
+            header_value="Token foobarbaz",
+            oas=f"{KOWNSL_API_ROOT}schema/openapi.yaml",
+        )
+
+        DefaultServiceFactory.create(
+            task_mapping=mapping,
+            service=cls.service,
+            alias="kownsl",
+        )
+
+        cls.task_dict = {
+            "topic_name": "some-topic",
+            "worker_id": "test-worker-id",
+            "task_id": "test-task-id",
+            "variables": {
+                "hoofdZaakUrl": {
+                    "type": "String",
+                    "value": "https://zaken.nl/api/v1/zaak/123",
+                    "valueInfo": {},
+                },
```

Comment:
you can/should use [`django_camunda.utils.serialize_variable`](https://github.com/maykinmedia/django-camunda/blob/master/django_camunda/utils.py#L69) for this instead

Reply:
for which part exactly?

Reply:
basically the entire build-up of the task variables, as it abstracts the boring Camunda internals:

```py
cls.task_dict = {
    "topic_name": "some-topic",
    "worker_id": "test-worker-id",
    "task_id": "test-task-id",
    "variables": {
        "hoofdZaakUrl": serialize_variable("https://zaken.nl/api/v1/zaak/123"),
        "reviewRequestId": serialize_variable("1"),
    }
}
```